templates/atom.xml

```diff
@@ -19,25 +20,14 @@
     <generator uri="https://www.getzola.org/">Zola</generator>
     <updated>{{ last_updated | date(format="%+") }}</updated>
     <id>{{ feed_url | safe }}</id>
+    {% set pageslen = pages | length %}
+    {% if pageslen > 15 %}
```

Comment:
Is this necessary? Have you tested what `end=15` does if there are fewer (or exactly) 15 pages?

Reply:
It seems needed, here's the output of `zola serve` without it

```
Error: Failed to serve the site
Error: Failed to render 'atom.xml'
Error: Reason: Variable `pages[i]` not found in context while rendering 'atom.xml': the evaluated version was `pages.1`. Maybe the index is out of bounds?
```